src/mon/MonCap.cc

```diff
@@ -218,6 +218,9 @@ void MonCapGrant::expand_profile_mon(const EntityName& name) const
     profile_grants.push_back(MonCapGrant("auth", MON_CAP_R | MON_CAP_X));
     profile_grants.push_back(MonCapGrant("config-key", MON_CAP_R | MON_CAP_W));
     profile_grants.push_back(MonCapGrant("config", MON_CAP_R | MON_CAP_W));
+    // ssh orchestrator provisions new daemon keys
+    profile_grants.push_back(MonCapGrant("auth get-or-create"));
```

Comment:
perhaps in the future the orchestrator could make a separate connection with a bootstrap profile that would be the union of existing bootstrap profiles? Then other parts of the mgr aren't effectively admins.